contracts/src/registry/IRegistryController.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.13;
+
+import {AccessControl} from "@openzeppelin/contracts/access/AccessControl.sol";
+
+interface IRegistryController {
```

Comment:
Most of this functionality looks like it's something the registrar would have rather than a controller? I'm not sure we need an interface for the controller at all, since it only calls the registrar, it doesn't get called by it.

Reply:
Apologies, this was supposed to be deleted.